accounts-db/src/accounts_db.rs

```diff
@@ -12561,40 +12561,36 @@ pub mod tests {
 
     #[test]
     fn test_hash_stored_account() {
-        // This test uses some UNSAFE tricks to detect most of hashing code changes, resulting from
-        // account's field additions and deletions of StoredAccountMeta and AccountSharedData and
-        // hashing-order changes.
-
+        // Number are just sequential.
+        let slot: Slot = 0x0102030405060708;
```

Comment:
nit: btw, you can place `_` arbitrarily: ie. `0x01_02_03_04_05_06_07_08`

Reply:
Added underscores.